docs/CactbotCustomization.md

```diff
@@ -86,6 +86,23 @@ The `oopsyraidsy` module loads `user/oopsyraidsy.js` and `user/oopsyraidsy.css`.
 And so on, for each module.
 These files are included after cactbot's files and can override its settings.
 
+Additionally, cactbot will also load any files in a subdirectory of the same module name.
+For example, `raidboss` will also load any `.js` and `.css` files
+underneath `user/raidboss/` even if in multiple subfolders.
+This is intended to make it easier to share triggers and customizations with others.
+Timeline `.txt` files must be directly in the same folder as the `.js` that refers to them.
+
+cactbot loads files in subdirectories (alphabetically) before loading files in outer directories.
+This is so that `user/raidboss.js` will always be loaded last
+and can override anything that is set inside a file inside of `user/raidboss/`.
+For example, `user/alphascape/some_file.js` will load before `user/mystatic/file1.js`,
```

Comment:
```suggestion
For example, `user/alphascape/some_file.js` will load before `user/mystatic/some_files.js`,
```

Reply:
I picked this specifically because `file1` sorts before `some_file`.  Is that confusing?  Is there a better way to capture both?